Framework/DataHandling/src/DataBlockComposite.cpp

```diff
@@ -11,8 +11,9 @@
 
 namespace {
 
-const int64_t invalidIntervalValue = std::numeric_limits<int64_t>::min();
-
+using Mantid::specnum_t;
+const specnum_t invalidIntervalValue = std::numeric_limits<specnum_t>::min();
```

Comment:
Can we follow our naming convention for "static" constants here, i.e all uppercase. At the point it is being read in the code `invalidIntervalValue` looks like a standard local variable. Uppercasing it means it's made clear to the reader this is a static constant.